### ISSUE TITLE
Support CUDA 10.2 ??

### ISSUE THREAD

**Issue by wangzihe1998** [gpgpu-sim/gpgpu-sim_distribution#218]
Hi,

Does GPGPU-sim support CUDA 10.2?
If operating under CUDA 10.2, do I need to change other configurations？

Thanks，
Zihe Wang

**Comment by jounghoolee** [comment]
As far as I've tried, it seems to wrok well with CUDA 10.0 11.2

**Comment by linloong** [comment]
So， doesn't support CUDA 10.2？

**Comment by esyvjlomh** [comment]
Still cannot use GPGPU-Sim with CUDA 10.2, with error like:
`lib/gcc-8.4.1/cuda-10020/release/libcudart.so.10.2: version'libcudart.so.10.2' not found`
Can I modify something to solve this problem?

**Comment by linloong** [comment]
Yeah, it's been a long time. If I remember correctly, I think you can directly link the needed `10.2.so` to the generated one, just like the examples below.

<img width="844" height="710" alt="Image" src="https://github.com/user-attachments/assets/f8a9ca7b-e0be-4d5d-8eed-f897405c92dc" />

**Comment by esyvjlomh** [comment]
> Yeah, it's been a long time. If I remember correctly, I think you can directly link the needed `10.2.so` to the generated one, just like the examples below.
> <img alt="Image" width="844" height="710" src="https://private-user-images.githubusercontent.com/39289129/494331658-f8a9ca7b-e0be-4d5d-8eed-f897405c92dc.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA0OTk3MjcsIm5iZiI6MTc2MDQ5OTQyNywicGF0aCI6Ii8zOTI4OTEyOS80OTQzMzE2NTgtZjhhOWNhN2ItZTBiZS00ZDVkLThlZWQtZjg5NzQwNWM5MmRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDE1VDAzMzcwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMwNjU3OWU5MTE3NzgwZWNiM2QzM2ZiZDU1MmQzYjJiYWViZmM1ZTY4NWRhM2M5YjA5YjkyMTYwYTgxYjVjNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UiwxVihpZtpOMJVqtc3EPKJCDfhoH9zkofcrhV5cpUo">

Thank you so much. I tried linking the file and it could work, but still have some API support problem (such as cudaMallocManaged and some Prefetch API). Have you met similar problem recently?
